config.go

```diff
@@ -1 +1,105 @@
 package main
+
+import (
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	flags "github.com/btcsuite/go-flags"
+	"path/filepath"
+	"io/ioutil"
+	"os"
+)
+
+const (
+	defaultConfigFilename = "lnwallet.conf"
+	defaultDataDirname    = "test_wal"
+	defaultRPCPort        = 10009
+	defaultSPVMode        = false
+	defaultPeerPort       = 10011
+	defaultBTCDHost       = "localhost:18334"
+	defaultBTCDUser       = "user"
+	defaultBTCDPass       = "passwd"
+	defaultBTCDCACertPath = ""
+	defaultUseRegtest     = false
+	defaultSPVHostAdr     = "localhost:18333"
+	defaultBTCDNoTLS      = false
+)
+
+var (
+	lnwalletHomeDir   = btcutil.AppDataDir("lnwallet", false)
```

Comment:
This line should instead be: 
```
lndHomeDir   = btcutil.AppDataDir("lnd", false)
```

The lines below referencing the previous variable name should also be updated.